### PR TITLE
linked to dicuss page

### LINKED ISSUE #8440: Help page still links to distutils-sig
**Describe the bug**
The [help page](https://pypi.org/help/) suggests:

> The Python Packaging Authority (PyPA) is a working group who work together to improve Python packaging. If you'd like to get in touch with a core packaging developer, use #pypa on IRC (freenode), or join the distutils-sig mailing list. 

though most of the `distutils-sig` discussion now occurs in [Discuss Packaging](https://discuss.python.org/c/packaging/14).

**Expected behavior**
Linking to the discuss instead seems possibly better if that's where most discussion/maintainers are participating.

---

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/).

If you are working on this issue and have questions, feel free to ask them here, in the [`#pypa-dev` chat channel on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or on the [distutils-sig.python.org mailing list](https://mail.python.org/mailman3/lists/distutils-sig.python.org/).

### PR DESCRIPTION
**Pull request by alpeshyadav**
Fixes #8440

### CODE REVIEW

**Comment by di** [comment]
Hi, thanks for working on this! I think the goal of #8440 was to change the `mailing_list_href`, not the `irc_href`. Our IRC channels haven't changed.

**Comment by alpeshyadav** [comment]
Oh, just a sec will change it

**Comment by alpeshyadav** [comment]
Why check failed?

**Comment by alpeshyadav** [comment]
Never worked on CI, got this error there. How to solve this?
`The command "make translations BINDIR="$(dirname $(which python))"" exited with 2.`

File: warehouse/templates/pages/help.html [hunk 1]
```diff
@@ -852,11 +852,11 @@ <h3>{% trans %}Resources{% endtrans %}</h3>
 
       <h3>{% trans %}Contact{% endtrans %}</h3>
       <p>
-        {% trans trimmed pypa_href='https://pypa.io', irc_href='https://webchat.freenode.net/?channels=%23pypa', mailing_list_href='https://mail.python.org/mailman/listinfo/distutils-sig', title=gettext('External link') %}
+        {% trans trimmed pypa_href='https://pypa.io', irc_href='https://webchat.freenode.net/?channels=%23pypa', discuss_href='https://discuss.python.org/c/packaging/14', title=gettext('External link') %}
           The <a href="{{ pypa_href }}" title="{{ title }}" target="_blank" rel="noopener">Python Packaging Authority (PyPA)</a>
           is a working group who work together to improve Python packaging. If you'd like to get in touch with a core packaging developer,
           use <a href="{{ irc_href }}" title="{{ title }}" target="_blank" rel="noopener">#pypa on IRC (freenode)</a>,
-          or <a href="{{ mailing_list_href }}" title="{{ title }}" target="_blank" rel="noopener">join the distutils-sig mailing list</a>.
+          or <a href="{{ discuss_href }}" title="{{ title }}" target="_blank" rel="noopener">#pypa on IRC (discuss)</a>.
```
Comment (ewjoachim) [inline warehouse/templates/pages/help.html:859]:
```suggestion
          or <a href="{{ discuss_href }}" title="{{ title }}" target="_blank" rel="noopener">browse the online board</a>.
```

**Comment by ewjoachim** [comment]
You have changed a translated string, so it's expected that you rebuild the translation file. This is explained in https://warehouse.pypa.io/development/getting-started.html#building-translations but I strongly advise you to read the document from the top :)

**Comment by alpeshyadav** [comment]
Sure

**Comment by alpeshyadav** [comment]
`/usr/bin/sh: .state/env/bin/python: No such file or directory`
What to do about this? Windows don't have a bin directory, switching to Linux distro is the only option, or something else?

**Comment by ewjoachim** [comment]
Honnestly, I don't know precisely. If I'm not mistaken, Windows can run bash now no ? https://en.m.wikipedia.org/wiki/Windows_Subsystem_for_Linux (we have left my expertise zone quite a bit...)

I can run the translations for you when I'm back to my computer at some point this week-end.

**Comment by alpeshyadav** [comment]
Ya, there was some setup issue with WSL, its resolved now. 
But I'm getting this error of module not found, where should I install it? 
` from babel.core import Locale
ModuleNotFoundError: No module named 'babel'`
This error popped when I ran `make translations` command

**Comment by alpeshyadav** [comment]
Finally resolved all errors!

**Comment by alpeshyadav** [comment]
Yayy, all tests passed! Will you merge it. Also please add the tag `hacktoberfest`, then only they will count it as a `valid PR`.

**Comment by alpeshyadav** [comment]
Can someone add `hacktoberfest-accepted` tag?

**Pull request merged**